### PR TITLE
Removes V1 annotations and clarify the values for Span::kind.

### PR DESCRIPTION
**Pull request by jcchavezs**
Ping @adriancole

### CODE REVIEW

File: src/Zipkin/Annotation.php [hunk 1]
```diff
@@ -7,45 +7,12 @@
...
-    const MESSAGE_RECV = 'mr';
-    const WIRE_SEND = 'ws';
-    const WIRE_RECV = 'wr';
-    const CLIENT_SEND_FRAGMENT = 'csf';
-    const CLIENT_RECV_FRAGMENT = 'crf';
-    const SERVER_SEND_FRAGMENT = 'ssf';
-    const SERVER_RECV_FRAGMENT = 'srf';
     const LOCAL_COMPONENT = 'lc';
     const ERROR = 'error';
     const CLIENT_ADDR = 'ca';
```
Comment (adriancole) [inline src/Zipkin/Annotation.php]: remove these

File: src/Zipkin/Annotation.php [hunk 2]
```diff
@@ -7,45 +7,12 @@
...
-
-    /**
-     * The client has received the response from the server. This sets the end of the span. The RPC is considered
-     * complete when this annotation is recorded.
-     */
-    const CLIENT_RECEIVE = 'cr';
-
-    const MESSAGE_SEND = 'ms';
-    const MESSAGE_RECV = 'mr';
-    const WIRE_SEND = 'ws';
```
Comment (adriancole) [inline src/Zipkin/Annotation.php]: you can keep wire send/receive

File: src/Zipkin/Annotation.php (hunk 1 above)
Comment (jcchavezs) [reply src/Zipkin/Annotation.php]: Only `CLIENT_ADDR`?

**Comment by jcchavezs** [comment]
like this @adriancole ?

File: src/Zipkin/Annotation.php [hunk 3]
```diff
@@ -7,45 +7,13 @@
...
     const WIRE_SEND = 'ws';
     const WIRE_RECV = 'wr';
-    const CLIENT_SEND_FRAGMENT = 'csf';
-    const CLIENT_RECV_FRAGMENT = 'crf';
-    const SERVER_SEND_FRAGMENT = 'ssf';
-    const SERVER_RECV_FRAGMENT = 'srf';
     const LOCAL_COMPONENT = 'lc';
     const ERROR = 'error';
-    const CLIENT_ADDR = 'ca';
     const SERVER_ADDR = 'sa';
```
Comment (adriancole) [inline src/Zipkin/Annotation.php]: sorry meant all the addrs. remoteEndpoint is based on span.kind now

**Pull request merged**